x/auth/spec/03_types.md

```diff
@@ -1,70 +0,0 @@
-<!--
-order: 4
--->
-
-# Types
```

Comment:
Are the corresponding new proto types documented elsewhere or will they be covered in a separate PR ?

Reply:
There's a "new page for Tx Signing" in #6953, I believe it suits more over there? It also echoes to Robert's refactor attempt to move tx stuff outside of x/auth.